README.md

```diff
@@ -209,59 +223,69 @@ FATAL   - Avoid empty password
 PASS    - Be unique UID
 PASS    - Be unique GROUP
 ```
+
 </details>
 
 ## Docker
 
-Replace [YOUR_CACHE_DIR] with the cache directory on your machine.
+Also, you can use Docker to use `dockle` command as follow.
 
+```bash
+$ docker run --rm goodwithtech/dockle [YOUR_IMAGE_NAME]
```

Comment:
I noticed that is better to use versioning images even if using docker.

```
$ VERSION=$(
 curl --silent "https://api.github.com/repos/goodwithtech/dockle/releases/latest" | \
 grep '"tag_name":' | \
 sed -E 's/.*"v([^"]+)".*/\1/' \
) && docker run --rm goodwithtech/dockle:${VERSION} [YOUR_IMAGE_NAME]
```

Reply:
> better to use versioning images even if using docker.

I understand.
But I think showing the simple version first, then following the versioned one "as a recommendation" would be better.

Since the purpose of Docker here, in the install section, is to let the user see how it works and easy like that. I feel it's losing the simplicity a little. Don't you think?